config/Makefile.in

```diff
@@ -580,6 +581,9 @@ DRIVER_SRCS+= \
 
 ICONS_CMXA=icons.cmxa
 
+LIBS_opt += $(OCAMLLIB)/threads/threads.cmxa
+LIBS_opt += $(CURL_PATH)/curl.cmxa -cclib "-L$(CURL_PATH) -lcurl-helper"
```

Comment:
we already depend on `ocamlfind` used in configure, so lets please use it for all new code
i will help to figure this out, but from top of my head it is `ocamfind query -i-format curl` for compiling and
`ocamlfind query -a-format -predicates native curl` and `ocamlfind query -l-format -predicates native curl` for linking